Formula/bats.rb

```diff
@@ -3,10 +3,11 @@ class Bats < Formula
   homepage "https://github.com/sstephenson/bats"
   url "https://github.com/sstephenson/bats/archive/v0.4.0.tar.gz"
   sha256 "480d8d64f1681eee78d1002527f3f06e1ac01e173b761bc73d0cf33f4dc1d8d7"
-  head "https://github.com/sstephenson/bats.git"
```

Comment:
Reason to remove ability to install --HEAD ?

Reply:
I re-added it